response_operations_ui/views/reporting_units.py

```diff
@@ -84,10 +84,7 @@ def edit_contact_details(ru_ref, respondent_id):
 
     ui_message = 'No updates were necessary'
     if 'emailAddress' in contact_details_changed:
-        if len(contact_details_changed) > 1:
-            ui_message = f'Contact details saved and verification email sent to {form.get("email")}'
-        else:
-            ui_message = f'Verification email sent to {form.get("email")}'
+        ui_message = f'Contact details changed and verification email sent to {form.get("email")}'
```

Comment:
This change break a acceptance test because of the upper and lower case V change. Put up a fix here: https://github.com/ONSdigital/rasrm-acceptance-tests/pull/117

Reply:
I'm writing some acceptance tests for this so I can put that in with them if you want?